src/layout/Sidebar.tsx

```
@@ -92,94 +94,132 @@ const Sidebar = () => {
-            >
-              v{APP_VERSION}-
-              {import.meta.env.VITE_GITHUB_HASH?.slice(0, 6)}
-            </button>
+          <OpenDrawerIcon className="size-5" />
+        </button>
+      )}
+      {isMobile && sidebarOpen && (
+        <div
+          className="fixed inset-0 z-30"
+          onClick={() => setSidebarOpen(false)}
+        ></div>
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)